pkgs/development/python-modules/ezyrb/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, future
+, numpy
+, scipy
+, matplotlib
+, scikit-learn
+, pytorch
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "ezyrb";
+  version = "1.3.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.6";
```

All Python releases  below 3.7 are EOL. The chance is very high that nobody will check if the packages work on Python < 3.7

```suggestion
  disabled = pythonOlder "3.7";
```